### DISCUSSION TITLE
An Automatic Docker image for RX 7000 series

### DISCUSSION THREAD

**Discussion by evshiron** [vladmandic/sdnext#891] [General]
The discussion mainly happened there:

* https://gist.github.com/In-line/c1225f05d5164a4be9b39de68e99ee2b?permalink_comment_id=4564587#gistcomment-4564587
* https://gist.github.com/In-line/c1225f05d5164a4be9b39de68e99ee2b?permalink_comment_id=4565415#gistcomment-4565415

Quote:

> The workflow and build script:
> https://github.com/evshiron/rocm_lab
> 
> The changes to vladmandic/automatic:
> https://github.com/evshiron/automatic/commit/fa52c83d052badfce6785e3b08680f5ae2b6ed30
> 
> The Docker image of vladmandic/automatic:
> `docker pull ghcr.io/evshiron/rocm_lab:rocm5.5-automatic`

There are limitations and I haven't seen a speed increase migrating from A1111 WebUI + Doggettx to Automatic + SDP, but at least it works pretty well.

**Comment by evshiron** [comment]
@vladmandic 

Greetings! I am here for some good news :)

Firstly, I wrote a tutorial for using [vladmandic/automatic](https://github.com/vladmandic/automatic) on RX 7000 series [here](https://evshiron.github.io/are-we-gfx1100-yet/post/automatic/) and it's quite easy.

Secondly, I managed to do PoC LoRA training with [kohya-ss/sd-scripts](https://github.com/kohya-ss/sd-scripts) on RX 7900 XTX, and I am looking forward to your CLI getting fixed (yesterday I tried https://github.com/vladmandic/automatic/commit/819d105d1d0a15d672b10c3f00a9c43dca9c01bf but it still said `duplicated lora name`). Please ping me if you get it working.

Finally, thank you for your work, now I have a kawaii waifu.

**Comment by vladmandic** [comment]
That is some great stuff!

I've just updated `train.py` script, but Lora/LyCORIS training is still broken - the problem is that lora training is not compatible with newer version of `diffusers` used here, it actually requires much older version.  
if you manually change `requirements.txt` to have `diffusers==0.14.0`, it will work.  
I don't want to downgrade `diffusers` globally for the entire repo, so I'm waiting for resolution of <https://github.com/kohya-ss/sd-scripts/issues/402>

**Reply by evshiron** [reply]
Yes. I followed your instructions and it just worked :)
